app/uk/gov/hmrc/gform/restapi/RestApiController.scala

```diff
@@ -14,23 +14,36 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.gform.config
+package uk.gov.hmrc.gform.restapi
```

Comment:
this is for fileupload at the moment, so I think the package name should reflect that